ui/apps/platform/src/Containers/ComplianceEnhanced/ScanConfigs/Wizard/useFormikScanConfig.tsx

```diff
@@ -0,0 +1,64 @@
+import { FormikProps, useFormik } from 'formik';
+import * as yup from 'yup';
+
+import { DayOfMonth, DayOfWeek } from 'Components/PatternFly/DayPickerDropdown';
+
+export type Parameters = {
```

Comment:
If another file imports this named export, it will come in with a generic name of "Parameters" unless aliased by the importing file.

```suggestion
export type ScanConfigParameters = {
```